src/component-library/Alert/Alert.style.tsx

```diff
@@ -12,14 +12,14 @@ interface StyledAlertProps {
 
 const StyledAlert = styled(Flex)<StyledAlertProps>`
   padding: ${theme.spacing.spacing2};
-  color: ${({ $status }) => theme.alert.status[$status]};
```

Comment:
Contrast was low with both bg and text colours set—spoke to Jay, and we've switched this to default text colours.